### ISSUE TITLE
[v2.8.0] command forseti scanner run is not working

### ISSUE THREAD

**Issue by joecheuk** [GoogleCloudPlatform/forseti-security#2251]
```
forseti scanner run

Argument error: Protocol message RunRequest has no "handle" field..
usage: forseti [-h] [--endpoint ENDPOINT] [--use_model USE_MODEL]
               [--out-format {json}]
               {explainer,inventory,config,model,scanner,notifier,server} ...

```

**Comment by averonesis** [comment]
actually this didn't helped, I've the same error on 2.8.0 @joecheuk

**Comment by joecheuk** [comment]
Hi @averonesis , I have merged the updates to release-2.8.0 branch and will port it back to dev once I finished the release testing.
